source/extensions/access_loggers/grpc/http_grpc_access_log_impl.cc

```diff
@@ -153,10 +152,10 @@ void HttpGrpcAccessLog::emitLog(const Http::RequestHeaderMap& request_headers,
 
     for (const auto& header : response_trailers_to_log_) {
       const auto entry = response_trailers.get(header);
```

Comment:
It seems that this is not needed anymore.

Reply:
yes. rm-ed